Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -81,6 +80,12 @@
 muonRPCDigis.InputLabel = 'rawDataCollector'
 castorDigis.InputLabel = 'rawDataCollector'
 
+#Add OnlineMetaData only from Run2_2018
+from Configuration.Eras.Modifier_run2_onlineMetaData_2018_cff import run2_onlineMetaData_2018
```

Comment:
summarizing the comments in this thread and during ORP meetings: the use case for which this development is thought, i.e. #27677, seems to be better served by customizing the consumer to use one or the other input, than the producer sequence as suggested here. 

An approach similar to that suggested here is in #27638, but to me that is a quite limited use case for a standalone online application, while for a general use in production workflows the suggestion by @makortel and @slava77 make more sense. 

I suggest to review the proposal along these lines.

Reply:
Let me add that if the `onlineMetaDataDigis` producer doesn't work for data/MC <= 2017, then, I think, it might be useful to add it to the Task with a Modifier so that the producer is available only in scenarios where it makes sense. Does `scalersRawToDigi` work for data/MC >= 2018?

(but regardless of that I still think that it would be better for the client side (as well) to customize the client configuration with a Modifier to consume only the needed product)